test/database/service/UserServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package database.service;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import database.dao.UserDataDAO;
+import database.dto.UserDataDTO;
+import service.UserService;
+
+@RunWith(JUnit4.class)
+public class UserServiceTest {
+	
+	@Mock
+	private UserDataDAO userDataDAO;	
+	private UserService userService;
+	
+	@Before 
+	public void setUp() {
+		MockitoAnnotations.openMocks(this);
+		userService = new UserService(userDataDAO);
+	}
+	
+	@Test
+	public void testLoginUser() {
+		UserDataDTO mockedUser = new UserDataDTO(1, "cbeltran", "goodpassword");
+		Mockito.doReturn(mockedUser).when(userDataDAO).getUserByLogin("cbeltran");
+		
+		boolean loginUser = userService.loginUser("cbeltran", "goodpassword");
+		Assert.assertTrue("Couln't login user", loginUser);
+	}
+	
+	@Test
+	public void testNoLoginUser() {
```

Comment:
For a future MR please add a test in which the user can't login due an incorrect password.